spec/integration/graphql_spec.rb

```diff
@@ -0,0 +1,235 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+if defined?(Rails)
+  enabled = true
+else
+  puts '[INFO] Skipping Rails spec'
+end
+
+if enabled
+  require 'active_record'
+  require 'action_controller/railtie'
+  require 'graphql'
+  require 'elastic_apm/graphql'
+
+  RSpec.describe 'GraphQL', :allow_running_agent, :spec_logger, :mock_intake do
+    include Rack::Test::Methods
+
+    def setup_database
+      ActiveRecord::Base.logger = Logger.new(SpecLogger)
+      ActiveRecord::Base.logger = Logger.new(nil)
+
+      ActiveRecord::Base.establish_connection(
+        adapter: 'sqlite3',
+        database: '/tmp/graphql.sqlite3'
+      )
```

Comment:
@estolfo perhaps we ought to add something like this to the generic Rails spec too?

Reply:
The `setup_database` method?

Reply:
Sure, yeah. Was mostly thinking about just having it include calling out to AR too. We're integration testing that in our own suite right now.

Reply:
Sure, that works.